irods_testing_environment/irods_config.py

```diff
@@ -16,9 +16,15 @@ def get_irods_zone_name(container):
 
 def get_irods_version(container):
     """Return the version of iRODS running on `container` as a tuple (major, minor, patch)."""
-    return tuple(int(i) for i in
-        json_utils.get_json_from_file(container, '/var/lib/irods/VERSION.json.dist')['irods_version']
-        .split('.'))
+    try:
+        return tuple(int(i) for i in
+            json_utils.get_json_from_file(container, '/var/lib/irods/version.json.dist')['irods_version']
+            .split('.'))
+
+    except:
+        return tuple(int(i) for i in
+            json_utils.get_json_from_file(container, '/var/lib/irods/VERSION.json.dist')['irods_version']
+            .split('.'))
```

Comment:
This could be a loop that returns on success.

Removes code duplication.

Thoughts?

Reply:
it's easy to read.   leave it.

Reply:
That does sound better to me. Let's do it

Reply:
Oh. Ninja'd. Leave it. :D